index.html

```diff
@@ -273,5 +274,49 @@ <h3>Skills</h3>
         <hr />
       </ul>
     </div>
+    <footer class="contact">
+      <section class="background2">
+        <form
+          class="form"
+          action="https://formspree.io/f/xdovrwzl"
+          method="POST"
+        >
+          <div class="section3">
+            <h2>Contact me</h2>
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it!
+            </p>
+            <input
+              id="name"
+              class="name"
+              name="name"
+              type="text"
+              placeholder="Lorem Ipsum"
+              maxlength="30"
+            />
+            <input
+              id="email"
+              class="email"
+              name="email"
+              type="email"
+              placeholder="loremipsum@gmail.com"
+            />
+            <textarea
+              id="text"
+              class="text"
+              type="text"
+              name="message"
+              placeholder="Write your message here..."
+              maxlength="500"
+            ></textarea>
+          </div>
+          <button type="submit" value="submit" class="submit">
+            Get in touch
+          </button>
```

Comment:
Great job on your button! 👏 However, there are a few improvements that can be made:

1. Hover, Active, and Disabled States: Currently, your button only has the default stage style. It's important to implement styles for the hover, active, and disabled states as per the design. This will provide visual feedback to users and ensure a consistent user experience. Please update your button to include these additional state styles. 🎨🔘
2. Your default stage styles also do not match the design perfectly. Please consider having a close look at the design and update your code accordingly.
![image](https://github.com/peter-mwovi/portfolio_setup/assets/55840999/9859a0e9-4233-4d77-b053-fcc058fb7c2d)

By incorporating these suggestions, you'll enhance the visual appeal and interactivity of your button, making it more in line with the design specifications. Keep up the excellent work! If you have any questions, feel free to ask. 🚀😊